### DISCUSSION TITLE
How can I get all devices that are subscribed to a specific topic?

### DISCUSSION THREAD

**Discussion by Eraldo** [xtrinch/fcm-django#263] [Q&A]
**Use case**
I want to show a notifications overview pages where the user can choose which topics to subscribe/unsubscribe.
For this to work I need to filter all devices associated with the user and get a list of all their subscribed topics, so that I can show the correct values on the settings page.

Is there a way to do this with the current models? Or do I have to keep track of subscribed and unsubscribed topics myself elsewhere?

**Comment by xtrinch** [comment]
There isn't, but honestly this would have been a really nice feature for this app - to have a model for topic connected to user model. If you are going to do this for your own app, please consider opening a PR within this package!

**Comment by xtrinch** [comment]
https://github.com/xtrinch/fcm-django/pull/302
This would be the proposal for the resolution of this discussion, lmk your thoughts
